packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -63,9 +63,7 @@ export class DocumentService implements api.IDocumentService {
         const historian = new Historian(
             this.gitUrl,
             true,
-            // Disable caching of storage requests until we implement non-URL-based caching
-            // when using WholeSummaries, because we cannot gurantee that a summary sha is unique to the document.
-            this.driverPolicies.enableWholeSummaryUpload,
```

Comment:
I think this is a driver side policy. Does it affect redis caching?

Reply:
Discussed with Tanvir offline - here are the details: when `disableCache` is enabled in the driver, it will include a query parameter with same name (`disableCache`) in the requests to Historian. In Historian (server side), every route for GET requests has a check like:
https://github.com/microsoft/FluidFramework/blob/72a22b97ea7dcaaa5f6c3f49c13b5408bfa2d7ab/server/historian/packages/historian-base/src/routes/summaries.ts#L57

That `useCache` parameter is passed to the `RestGitService` and it eventually ends up being used in the `resolve()` function. If `useCache` is false (due to `disableCache` being true, coming from the driver), we ignore the cache:

https://github.com/microsoft/FluidFramework/blob/72a22b97ea7dcaaa5f6c3f49c13b5408bfa2d7ab/server/historian/packages/historian-base/src/services/restGitService.ts#L395

However, the `useCache` parameter derived from the query string is also used in `handleResponse()` under utils.ts:

https://github.com/microsoft/FluidFramework/blob/aabc246ef86ed993b5a098c2c16b84b6e35baf1b/server/historian/packages/historian-base/src/routes/utils.ts#L16-L29

So making `disableCache` as `false` in the driver would accidentally enable browser caching if nothing else is done to prevent it. We don't want browser caching for summary data yet due to some incompatibilities with how whole summary works. To fix that, I'm also making the GET Summary route in summaries.ts hardcode `false` for `useCache` passed to `handleResponse`, so we always set the headers correctly.